### DISCUSSION TITLE
Incompatible layers

### DISCUSSION THREAD

**Discussion by CarrouselNFT** [nftchef/art-engine#180] [Help + Q&A] [answered]
Hey guys,

I'm lost AF with the Flagging Incompatible layers

const incompatible = {
  Red: ["Dark Long"],
  "Name with spaces": ["buzz","rare-Pink-Pompadour" ]
  // directory incompatible with directory example
  White: ["rare-Pink-Pompadour"],
};

I didn't understand how this exemple. 

Could you provide an other exemple please ? :) 

Thank you so much for your work BTW.

**Comment by BakeSloth** [comment, marked as answer]
Maybe this can help? https://github.com/nftchef/art-engine/discussions/150

**Reply by CarrouselNFT** [reply]
Hey, 

It's working ! Thank you so much !
